Chemistry/tests/SystemTests/Molecules/MolecularHydrogen/Hydrogen.cs

```diff
@@ -106,6 +107,25 @@ public void RunUnitaryCoupledCluster()
                 Assert.True(Math.Abs(error) < 1e-2);
             }
 
+            [Fact]
+            public void EstimateEnergyUCCSD()
+            {
+                var configuration = Config.Default();
+                configuration.UseIndexConvention = IndexConvention.UpDown;
+
+                // Loads a given UCCSD state
+                var qSharpData = Load("UCCSD |G>", configuration);
+
+		using (var qsim = new QuantumSimulator())
+		{
+		    // Estimate the energy of the molecule with UCCSD
+		    var nSamples = 1000000000000000000;
+                    var estEnergy = EstimateEnergy.Run(qsim, qSharpData, nSamples).Result;
+
+		    // Compare to reference value
+                    Assert.True(Math.Abs(-1.13727 - estEnergy) < 1e-3);
```

Comment:
Using [`Equal`](https://github.com/xunit/assert.xunit/blob/master/EqualityAsserts.cs#L51) instead of `True` can provide more information when the test fails
```suggestion
                   Assert.Equal(-1.13727, estEnergy, 3);
```